identity/app/form/PrivacyMapping.scala

```diff
@@ -120,5 +149,6 @@ object PrivacyFormData {
       receiveGnmMarketing = userDO.statusFields.receiveGnmMarketing,
       receive3rdPartyMarketing = userDO.statusFields.receive3rdPartyMarketing,
       allowThirdPartyProfiling = userDO.statusFields.allowThirdPartyProfiling,
-      consents = if (userDO.consents.isEmpty) defaultConsents else userDO.consents)
+      consents = if (userDO.consents.isEmpty) defaultConsents else userDO.consents.filter(checkIfConsentExistsInModel)
```

Comment:
@mario-galic if i'm reading this correctly, a user with any set consents will never read from `defaultConsents` again. What is ensuring they can get any potential new consents from `identity`?